units/en/unitbonus1/train-play.mdx

```diff
@@ -0,0 +1,15 @@
+# Let's train and play with Huggy 🐶 [[train-play]]
+
+## Let's train Huggy 🐶
+
+Open this colab:
+
+
+## Play with Huggy
+Now that you've trained Huggy and pushed it to the Hub. **You will be able to play with him ❤️**
+
+Just launch this game:
+
+CLic
+
+ADD STEPS
```

Comment:
Not sure what the plan is here, one option is to dynamically load the model, another option is to duplicate your space and modify the model id

Reply:
It's the first one, the second one is technically not possible for now given the fact that people will need to compile the game using Unity and change the model inside the Unity Editor which adds a lot of friction.